build_tools/waf_dynamo.py

```diff
@@ -1589,10 +1589,6 @@ def detect(conf):
         conf.env['STATICLIB_LUA'] = 'lua'
     else:
         conf.env['STATICLIB_LUA'] = 'luajit-5.1'
-        if '64' in build_util.get_target_platform():
-            conf.env['CXXDEFINES_LUA'] = ['LUA_BYTECODE_ENABLE_64']
-        else:
-            conf.env['CXXDEFINES_LUA'] = ['LUA_BYTECODE_ENABLE_32']
```

Comment:
Also note the `LUA_BYTECODE_ENABLE_64` is also specified in build.yml.

Reply:
Ah, thanks, I've removed those defines as well now.